### PR TITLE
Feature/20648 auth implementation

### PR DESCRIPTION
**Pull request by istherepie**
Minor changes to auth handler. 
Removed username from URI request path
Username is passed as part of the json payload

### CODE REVIEW

File: mora/auth.py [hunk 1]
```diff
@@ -108,13 +111,12 @@ def login(username):
     return resp
 
 
-@blueprint.route('/service/user/<user>/logout', methods=['POST'])
+@blueprint.route('/service/user/logout', methods=['POST'])
 def logout(user=None):
```
Comment (danchr) [inline mora/auth.py]: Please remove the parameter as well.

**Comment by istherepie** [comment]
Requested changes fixed.

**Comment by danchr** [comment]
Oh, and it would be great if we could move the endpoint away from `/mo` — that's legacy cruft too. The blueprint prefix should just be `/service`.

**Comment by istherepie** [comment]
Fixed as requested.

**Comment by danchr** [comment]
Awesome, thanks!

**Pull request merged**